homeassistant/components/thermopro/sensor.py

```diff
@@ -59,6 +59,15 @@
         state_class=SensorStateClass.MEASUREMENT,
         entity_registry_enabled_default=False,
     ),
+    (
+        ThermoProSensorDeviceClass.BATTERY,
+        Units.PERCENTAGE,
+    ): SensorEntityDescription(
```

Comment:
Should this be entity category diagnostic ?

Reply:
ecd48485c20